setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 setup(
     name='orchestra',
-    version='0.1',
+    version='0.1.1',
```

Comment:
Current master is at `0.1.1` (which is also why this PR isn't mergeable). You'll want to update to `0.1.2`.